### PR TITLE
Remove calls to `ad-m/github-push-action`

### PR DESCRIPTION
**Pull request by gastaldi**
- No need to call it again as tags are already pushed in the previous call

### CODE REVIEW

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml [hunk 1]
```diff
@@ -77,10 +77,3 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: ${{github.base_ref}}
-
-      - name: Push tags
-        uses: ad-m/github-push-action@v0.6.0
```
Comment (gsmet) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:82]: Couldn't we get rid of it entirely? My understanding was that with the new version of `actions/checkout`, we would be able to push with `git` directly?

Comment (gastaldi) [reply independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:82]: Hm, maybe yes, I'll need to test before doing that

**Comment by gastaldi** [comment]
Marking as a draft until I test it properly

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
           git checkout -b release
-          mvn -B release:prepare -Prelease -DpreparationGoals="clean install" -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
+          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
```
Comment (gsmet) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:50]: This looks unrelated? Also I thought it was added because of the codestarts?

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml [hunk 3]
```diff
@@ -73,14 +73,5 @@ jobs:
...
-          branch: ${{github.base_ref}}
-
-      - name: Push tags
-        uses: ad-m/github-push-action@v0.6.0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          tags: true
-          branch: ${{github.base_ref}}
+        run: |
+          git push origin ${{github.base_ref}} --tags
```
Comment (gsmet) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:77]: This looks good to me.

**Review by gsmet — COMMENTED** [review]
Added a question about a weird change.

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml (hunk 2 above)
Comment (gsmet) [reply independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:50]: Ah OK, thanks for the clarification :)

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml (hunk 1 above)
Comment (gastaldi) [reply independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml:82]: Done

**Review by gsmet — APPROVED** [review]
